terraform/aws/public-cloud/main.tf

```diff
@@ -1,8 +1,19 @@
 variable "access_key" {}
 variable "secret_key" {}
 variable "organization" { default = "kubeform" }
-variable "region" { default = "eu-west-1" }
-variable "availability_zones" { default = "eu-west-1a,eu-west-1b,eu-west-1c" }
+#variable "region" { default = "eu-west-1" }
+#variable "availability_zones" { default = "eu-west-1a,eu-west-1b,eu-west-1c" }
```

Comment:
I should have used a tfvars file instead of doing it this way.
I thought us-west-2 had only 2 regions, (like us-west-1), but I just found out it has 3.